src/sentry/sentry_metrics/querying/api.py

```diff
@@ -479,7 +504,7 @@ def run_metrics_query(
 
     # Parsing the input and iterating over each timeseries.
     parser = QueryParser(fields=fields, query=query, group_bys=group_bys)
-    for timeseries in parser.parse_timeserieses():
+    for timeseries in parser.generate_queries(environments=environments):
```

Comment:
This is a bit confusing now as it returns instances of `Timeseries`, but it mentions queries where an actual `Query` type exists. Would the former naming of `parse_timeseries` (see spelling) or `generate_timeseries` be better?

Reply:
Yeah, good point